diff --git a/node_modules/source-map-resolve/index.js b/node_modules/source-map-resolve/index.js
--- a/node_modules/source-map-resolve/index.js
+++ b/node_modules/source-map-resolve/index.js
@@ -1,7 +1,7 @@
 var atob = require("atob")
 var urlLib = require("url")
 var pathLib = require("path")
-var decodeUriComponentLib = require("decode-uri-component")
+var decodeUriComponentLib = require("decode-uri-component").default
 
 
 
