diff --git a/node_modules/svgo-browser/lib/svgo/jsAPI.js b/node_modules/svgo-browser/lib/svgo/jsAPI.js
--- a/node_modules/svgo-browser/lib/svgo/jsAPI.js
+++ b/node_modules/svgo-browser/lib/svgo/jsAPI.js
@@ -327,7 +327,7 @@ JSAPI.prototype.someAttr = function (callback, context) {
  * @return {Array} null if no elements matched
  */
 JSAPI.prototype.querySelectorAll = function (selectors) {
-  const matchedEls = cssSelect(selectors, this, cssSelectOpts);
+  const matchedEls = cssSelect.selectAll(selectors, this, cssSelectOpts);
 
   return matchedEls.length > 0 ? matchedEls : null;
 };
